src/components/CountriesLink.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from "react";
+import { CountryLink } from "@components/CountryLink";
+import { getCountryName } from "@helpers/getCountryFields";
+import { isSystemGeo } from "@utils/isSystemGeo";
+import { TGeography } from "@types";
+
+type TCountriesLink = {
+  geographies: string[];
+  countries: TGeography[];
+};
+
+export const CountriesLink: FC<TCountriesLink> = ({ geographies, countries }) => (
```

Comment:
This component doesn't need to be a FunctionalComponent (and I think this is a legacy feature of react now anyway, reserved for when we need to use `children` as a prop)

Reply:
Ahh okay - my bad blindly copying from other parts of the codebase! Thanks for letting me know. Think I've updated accordinly - but let me know if I've still got it wrong